README.md

```diff
@@ -80,6 +80,14 @@ You can see a number of examples [here](examples).
 | from_share                  | map    | Specify a provider and a share in this map to create a database from a share. | true     | false     | false    | <nil>   |
 | name                        | string |                                                                               | false    | true      | false    | <nil>   |
 
+#### import
```

Comment:
I like what you are doing here, but all the docs between `<!-- START -->` and `<!-- END -->` is auto-generated  see the [docs make target](https://github.com/chanzuckerberg/terraform-provider-snowflake/blob/4346dee4430764fa7c608a761b5ffd3c53650631/Makefile#L94).

Reply:
oops forgot about that, will look into that later.